config_src/nuopc_driver/mom_cap.F90

```diff
@@ -606,6 +606,21 @@ subroutine InitializeAdvertise(gcomp, importState, exportState, clock, rc)
   Ice_ocean_boundary%lrunoff         = 0.0
   Ice_ocean_boundary%frunoff         = 0.0
 
+  Ice_ocean_boundary%use_waves=ocean_state%use_waves
```

Comment:
I don't see where this variable is declared but it must be, otherwise the implicit none should catch it. 

It seems it is only used in mom_cap_methods to allocate and fill the import state. You could probably do this w/o having to define ice_ocean_boundary%use_waves if you checked instead for the ustkb,vstkb being allocated in ice_ocean_boundary.

Reply:
The use_waves was added the Ice_ocean_boundary type.  You are correct that we can check if ustkb is allocated instead.  I am running the test with this final update and then will push that plus the logs up.